pycaribbean-2016/videos/pycaribbean-2016-how-to-teach-git-by-georgia-reh.json

```
@@ -9,6 +9,10 @@
     "https://recompilermag.com/issues/issue-1/how-to-teach-git/",
     "http://georgiareh.com/2016/02/pycaribbean-how-to-teach-git"
   ],
+  "useful links": [
```

```suggestion
  "related_urls": [
```

See the schema here: https://github.com/pyvideo/data/blob/c5715777697d80d88a2b35fc4d167417e668906f/.schemas/video.json#L46-L69

---

Thank you very much!